course/src/main/java/in/testpress/course/ui/viewholders/ContentListItemViewHolder.kt

```diff
@@ -20,4 +16,13 @@ class ContentListItemViewHolder(view: View) : BaseContentListItemViewHolder(view
         }
 
     }
-}
\ No newline at end of file
+
+    override fun bindContentDetails(content: DomainContent) {
+        super.bindContentDetails(content)
+        when (content.contentTypeEnum) {
+            ContentType.Notes -> contentIcon.setImageResource(R.drawable.writing)
+            ContentType.Attachment -> contentIcon.setImageResource(R.drawable.paperclip)
+            else -> contentIcon.setImageResource(R.drawable.ic_live)
```

Comment:
what about other content types ? video, exam

Reply:
For video and exam we have seperate viewholder